internal/pkg/cli/job_init.go

```diff
@@ -336,17 +336,18 @@ func (o *initJobOpts) legitimizePlatform() error {
 		return fmt.Errorf("get docker engine platform: %w", err)
 	}
 	detectedPlatform := dockerengine.PlatformString(detectedOs, detectedArch)
-	platform, err := manifest.RedirectPlatform(detectedOs, detectedArch, o.wkldType)
+	redirectedPlatform, err := manifest.RedirectPlatform(detectedOs, detectedArch, o.wkldType)
 	if err != nil {
 		return fmt.Errorf("redirect docker engine platform: %w", err)
 	}
-	if platform == "" {
+	if redirectedPlatform == "" {
 		return nil
 	}
-	if platform != detectedPlatform {
-		log.Warningf("Your platform %s is currently unsupported. Setting platform to %s in your manifest.\n", detectedPlatform, platform)
-		o.platform = &platform
+	if redirectedPlatform != detectedPlatform {
+		log.Warningf("Your architecture type is currently unsupported. Setting platform %s instead.\n", redirectedPlatform)
 	}
+	log.Warningf("See 'platform' field in your manifest.\n")
```

Comment:
nit: what do you think of?
```suggestion
		log.Warningf("Your architecture type %s is currently unsupported.\n", color.HighlightCode(detectedPlatform))
		log.Warningf("Setting %s instead in your manifest.\n", colorHighlightCode(fmt.Sprintf("platform: '%s'", redirectedPlatform))
	}
	
```

Reply:
I like it! Combining into one log line (but with \n) so "NOTE:" doesn't appear twice, and also using just `detectedArch`. Also not using the single quotes for the platform bc we don't in the manifest.